components/StoryPoC/VideoScrollStory.tsx

```diff
@@ -0,0 +1,127 @@
+'use client';
+import { useRef } from 'react';
+import { useScroll, m, useTransform } from 'framer-motion';
+import { scroll } from 'framer-motion/dom';
+import { AnimateInView } from '../Animate';
+import { Container } from '../Container';
+import { Grid } from '../Grid';
+import { Heading, Paragraph, Text } from '../Typography';
+import { getProcessedImage } from '@/utilities/getProcessedImage';
+import { getMaskedAsset } from '@/utilities/getMaskedAsset';
+import { colorNameToHex } from '@/utilities/colorPalettePlugin';
+import { Paragraphs } from '../Temporary/Paragraphs';
+
+export const VideoScrollStory = () => {
+  const contentRef = useRef<HTMLDivElement>(null);
+  const { scrollYProgress } = useScroll({
+    target: contentRef,
+  });
+  const robinsEgg = colorNameToHex['robins-egg'];
+  const lime = colorNameToHex.lime;
+  const lavender = colorNameToHex.lavender;
+  const fuchsia = colorNameToHex.fuchsia;
+  const plumLight = colorNameToHex['plum-light'];
+  const colorChange = useTransform(scrollYProgress, [0, 0.33, 0.67, 1], [robinsEgg, lime, lavender, plumLight]);
+
+  return (
+    <Container width="full" className="relative" bgColor="black">
+      <div className="sticky top-0 h-screen w-full">
+        <video
+          playsInline
+          autoPlay
+          preload="auto"
+          muted
+          loop
+          aria-label="Background Video"
+          className="block w-full h-full object-cover"
+        >
+          <source src={getMaskedAsset('https://a-us.storyblok.com/f/1005200/x/80ddd8341f/starloop.webm')} type="video/webm" />
+          <source src={getMaskedAsset('https://a-us.storyblok.com/f/1005200/x/1a881ffc0a/starloop.mp4')} type="video/mp4" />
+          <p>Your browser does not support HTML video.</p>
+        </video>
+        <m.div
+          style={{ backgroundColor: colorChange }}
+          className="absolute top-0 r-0 w-full h-full mix-blend-overlay"
+        />
+        <div className="absolute top-0 r-0 w-full h-full bg-gc-black mix-blend-soft-light" />
```

Comment:
These are the 2 overlays on top of the background video. One changes color on scroll from blue -> lime -> lavender -> darker purple with a blend mode of "overlay" (kind of like the photoshop effects). The other one is a black layer with the "soft light" blend mode.